deep_qa/common/util.py

```diff
@@ -30,3 +30,42 @@ def add_noise_to_dict_values(dictionary: Dict[Any, float], noise_param: float) -
         noise = random.uniform(-noise_value, noise_value)
         new_dict[key] = value + noise
     return new_dict
+
+
+def clean_layer_name(input_name: str,
+                     keep_right_of_last_backslash: bool=None,
```

Comment:
This name is confusing to me.  Seems like you want a parameter named "split on backslash", or "remove nested scopes", or something, with only one behavior - this is never called with `True`, only with `False` and `None`, so we can simplify this and name it something more appropriate.